### DISCUSSION TITLE
Wix like comments widget

### DISCUSSION THREAD

**Discussion by Mulli** [elementor/elementor#22656] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Clients moving their website from Wix to Wordpress/Elementor ask for the same comment widget.

This is currently impossible to achieve using Elementor comments widget
and requires coding.

### Describe the solution you'd like

Feature rich comments widget

### Describe alternatives you've considered

It requires a lot of coding. Will not get into it.
Its a developer team task.

**Comment by jazir555** [comment]
I think a markup.io style implementation would be preferable to solve the issue you are referring to, as well as pairing it with the 2 issues listed here:

https://github.com/orgs/elementor/discussions/22811

https://github.com/orgs/elementor/discussions/22840

These 3 are best linked together IMO into the one single full implementation allowing all 3 issues to be implemented/satisfied simultaneously in my opinion that can resolved all in one go, and would solidify all 3 issues into one large fix. 

Apologies for the redundancy in the past paragraph, I am not sure how to best phrase my comment here in a more concise way atm.
